docs/news.rst

```diff
@@ -60,6 +60,10 @@ Scrapy 2.5.1 (2021-10-05)
     need to upgrade scrapy-splash to a greater version for it to continue to
     work.
 
+*   **Deprecations:**
```

Comment:
This adds a note to the 2.5.1 section which is not true. And in general e don't add new entries with the changes themselves, only doing that when preparing a release. So it's fine to omit it.